flocker_bb/steps.py

```diff
@@ -22,6 +23,27 @@
 
 TWISTED_GIT = b'https://github.com/twisted/twisted'
 
+flockerBranch = Interpolate("%(src:flocker:branch)s")
+
+buildNumber = Interpolate("%(prop:buildNumber)s")
```

Comment:
This looked a bit too global. Is `Interpolate` re-evaluated at each build? I guess it must be.